app/frontend/src/components/forms/submission/StatusPanel.vue

```diff
@@ -403,14 +403,14 @@ export default {
               >
                 <!-- selected user -->
                 <template #chip="{ props, item }">
-                  <v-chip v-bind="props" :text="item?.raw?.fullName" />
+                  <v-chip v-bind="props" :text="item?.fullName" />
                 </template>
                 <!-- users found in dropdown -->
                 <template #item="{ props, item }">
                   <v-list-item
                     v-bind="props"
-                    :title="`${item?.raw?.fullName} (${item?.raw?.email})`"
-                    :subtitle="`${item?.raw?.username} (${item?.raw?.idpCode})`"
+                    :title="`${item?.fullName} (${item?.email})`"
+                    :subtitle="`${item?.username} (${item?.idpCode})`"
```

Comment:
Similar to the Team Management page, these are displaying as "undefined" (possibly the same underlying problem?)
![image](https://github.com/bcgov/common-hosted-form-service/assets/35532993/6b91cf34-5265-4302-85df-afdb456797a3)

Reply:
This should be resolved now